pkg/api/persistentvolume/util.go

```diff
@@ -49,12 +56,33 @@ func GetWarningsForPersistentVolume(pv *api.PersistentVolume) []string {
 	if pv == nil {
 		return nil
 	}
-	return warningsForPersistentVolumeSpecAndMeta(nil, &pv.Spec)
+	return warningsForPersistentVolumeSpecAndMeta(nil, &pv.Spec, &pv.ObjectMeta)
+}
+
+func GetWarningsForStorageClassAnnotations(fieldPath *field.Path, meta *metav1.ObjectMeta) []string {
```

Comment:
exporting a method from the persistentvolume package and reusing it from the persistentvolumeclaim package is a little weird... I would probably just replicate/inline the annotation check and warning in `warningsForPersistentVolumeSpecAndMeta` and `GetWarningsForPersistentVolumeClaim`

Reply:
The reason I add the exported function is because maybe we need it somewhere else, but I didn't find a suitable pkg to add it.
Yes it's a little weird, so let's remove exported method and add to both functions